### DISCUSSION TITLE
Is Undertow needed?

### DISCUSSION THREAD

**Discussion by agoncal** [jhipster/generator-jhipster-quarkus#208] [Q&A]
There is an Undertow dependency in the `pom.xml`: 

```
        <dependency>
            <groupId>io.quarkus</groupId>
            <artifactId>quarkus-undertow</artifactId>
        </dependency>
```

I'm wondering if this is still needed ?

**Comment by danielpetisme** [comment]
Undertow has been introduced to implement this filter
https://github.com/jhipster/jhipster-sample-app-quarkus/blob/main/src/main/java/io/github/jhipster/sample/web/rest/SpaFilter.java

The filter is required when having smart web UI (like angular)  as explained here
https://quarkus.io/blog/quarkus-and-web-ui-development-mode/


I would love to see Undertow removed. The filter as to be re-implemented in pure Vert.x.
